### DISCUSSION TITLE
Be able to Compare two different DKG

### DISCUSSION THREAD

**Discussion by Python700** [MKT-Playbook/Hub#30] [Feature requests]
Hello, first off thank you for making this amazing tool you’ve made for MKT!
Based off of the title, I’d like a feature to where you can Compare two DKG. For example, say I’d like to compare Gliders A and B. It will compare A and B and if there are any overlaps, the track will be marked RED. So if A and B shares Ninja Hideaway, Ninja Hideaway will be marked RED. This would be useful if you decide if you want to Lvl Up a DKG and want to make sure it doesn’t overlap any other powerful DKG.

**Comment by TitusPosiadly** [comment]
Hey, thanks for the suggestion and the message.
Love the idea! I had something similar planned a while ago but pushed it back in priority in order to finally get the performance issues under control. I also decided to first add a coverage tool as the next big addition, but I might be able to add something along the lines of what you are looking for in there as well. That being said a tool to compare different cards is something I would like to add at some point as well. Also love your idea with the marked tracks - will keep that in mind once I start working on it.
